src/views/modal_handler.rs

```diff
@@ -120,13 +136,22 @@ impl DisplayModal {
             // Display piggui information
             ModalMessage::VersionModal => {
                 self.show_modal = true;
+                self.is_warning = false;
                 self.modal_type = Some(ModalType::Info {
                     title: "About Piggui".to_string(),
                     body: crate::views::version::version().to_string(),
+                    is_version: true,
                 });
                 Command::none()
             }
 
+            ModalMessage::OpenRepoLink => {
```

Comment:
It's pretty poor that Iced (or Iced_aw) doesn't let you put hypertext in a text in a dialog and handle the links for you, and we have to do all this ... :-( 

Consider a PR to them to improve this as a personal contribution?